src/Hystrix.Dotnet.AspNet/HystrixDefaultConfigurationElement.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Configuration;
+
+namespace Hystrix.Dotnet.AspNet
+{
+    public class HystrixDefaultConfigurationElement : ConfigurationElement
+    {
+        [ConfigurationProperty("commandTimeoutInMilliseconds", IsRequired = false, DefaultValue = 1000)]
```

Comment:
In our remote config we currently have a default value of `60000`. All the other defaults already match the remote defaults.

Reply:
Yeah, I'd also prefer to changing it to `60000`, this `1000` is too low for most use cases.

Are you okay with changing it without bumping the major version of the library?

Reply:
Yep